### DISCUSSION TITLE
A note of appreciation

### DISCUSSION THREAD

**Discussion by shakfu** [DBraun/ChucKDesigner#15] [Show and tell]
Hi @DBraun thanks for sharing this project.

I found it to be extremely useful in developing my own [max/msp chuck external](https://github.com/shakfu/chuck-max).

👍

**Comment by DBraun** [comment]
Awesome! I'll share with some chuck friends.

**Reply by shakfu** [reply]
Thanks, hope they enjoy it! 😄
